sdk/evaluation/azure-ai-evaluation/tests/e2etests/test_lite_management_client.py

```diff
@@ -31,7 +31,7 @@ def test_get_token(self, project_scope, azure_cred):
         )
 
         token = client.get_token()
-        assert isinstance(token, AccessToken) and len(token) > 0
+        assert isinstance(token, str) and len(token) > 0
```

Comment:
Same here we should follow this protocol for get_token https://learn.microsoft.com/en-us/python/api/azure-core/azure.core.credentials.tokencredential?view=azure-python#azure-core-credentials-tokencredential-get-token